docs/config.json

```
@@ -435,7 +440,7 @@
               "title": "Discord",
               "slug": "/access-controls/access-request-plugins/ssh-approval-discord/",
               "forScopes": ["enterprise", "cloud"]
-            }
+            },
```

You don't need this trailing comma. I don't think it will even compile if you keep it.
```suggestion
            }
```